src/main/webapp/WEB-INF/views/Car/all.jsp

```diff
@@ -0,0 +1,37 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>GET ALL CARS</title>
+</head>
+<body>
+<h1>List of cars:</h1>
+<table>
+    <tr>
+        <td>ID</td>
+        <td>MODEL</td>
+        <td>MANUFACTURE ID</td>
+        <td>MANUFACTURE NAME</td>
+        <td>MANUFACTURER COUNTRY</td>
+        <td>DRIVERS</td>
+        <td>DELETE</td>
+    </tr>
+    <c:forEach items="${cars}" var = "car">
+        <tr>
+            <td><c:out value="${car.id}"/></td>
+            <td><c:out value="${car.model}"/></td>
+            <td><c:out value="${car.manufacturer.id}"/></td>
+            <td><c:out value="${car.manufacturer.name}"/></td>
+            <td><c:out value="${car.manufacturer.country}"/></td>
+            <td>
+                <c:forEach items="${car.drivers}" var="driver">
+            <td><c:out value="${driver.id}"/></td>
+            <td><c:out value="${driver.name}"/></td>
+            </c:forEach>
+            </td>
+            <td><a href="${pageContext.request.contextPath}/cars/delete?id=${car.id}"> delete </a></td>
+        </tr>
+    </c:forEach>
+</table>
+</body>
+</html>
```

Comment:
Please add an empty line to the end of the file. Fix the same issue in all places